### PR TITLE
Update README.md: Add Scramjet Cloud Platform

### PR DESCRIPTION
**Pull request by MichalCz**
### Scramjet Cloud Platform

Hi, I'm adding Scramjet Cloud Platform - a SaaS serverless stream processor that allows acquiring, transforming and consuming data - so potentially running the whole data pipeline on it's own as an end-to-end data transform tool. It's in free beta, but it does add a hefty 15 PB*s year of free tier to be used over a year.

### New Submission Check List

Please ensure your submission ticks all of these boxes:

 - [x] This is Software as a Service not self hosted
 - [x] It has a free tier not just a free trial
 - [x] The submission mentions what is free
 - [x] The submission is not already present in the list

**Thank You, this is a great list! I'd be glad if our software would be a part of it**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -640,7 +640,7 @@ Table of Contents
   * [pubnub.com](https://www.pubnub.com/) - Swift, Kotlin and React messaging at 1 million transactions each month. Transactions may contain multiple messages.
   * [eyeson API](https://developers.eyeson.team/) - A video communication API service based on WebRTC (SFU, MCU) to build video platforms. Allows RealTime Data Injection, Video Layouts, Recordings, a fully featured hosted web UI (quickstart) or packages for custom UIs. Has a [free tier for developers](https://apiservice.eyeson.com/api-pricing) with 1000 meeting minutes a month.
   * [Upstash Kafka](https://upstash.com/kafka) - Serverless Kafka Cloud offering with per-request-pricing. It has a free tier with max 10000 messages per day.
-
+  * [Scramjet Cloud Platform Beta](https://www.scramjet.org/#join-beta) - An end-to-end stream processing platform in free beta and offering 15 petabyte-seconds of free compute after the beta ends.
```
Comment (ripienaar) [inline README.md:643]: I think to add such a beta program we need to know details of its eventual pricing, things tend to get orphaned on this list, so adding a beta here where the benefits are known only to apply to beta users would just leed to stale data soon.

Comment (MichalCz) [reply README.md:643]: Should I add this to the wording?

Comment (MichalCz) [reply README.md:643]: https://docs.scramjet.org/platform/pricing - this is the pricing calculator on our page. It reflects the above pricing exactly.

If you have any remarks though, I'd be happy to implement. We're very early stage, so all feedback is really needed. :)

Comment (ripienaar) [reply README.md:643]: OK, seems ok but looks like there is a conflict in the branch, do you mind rebasing?

Comment (MichalCz) [reply README.md:643]: No probs. :)

Comment (MichalCz) [reply README.md:651]: Ready. :)

**Pull request merged**